source/includes/Core/_payout.md

```diff
@@ -143,6 +144,7 @@ user_api_token | string | optional (either recipient or user_api_token required)
 currency | string | optional | 3-letter ISO code for currency | Default to 'SGD'
 descriptions | string | optional | A short description for this payout. This will be part of the email/SMS that the recipient will be receiving from Xfers. | Payment for Rent for July
 wallet_id | integer | optional | Coming Soon | Default to XFERS Wallet
+meta_data | string | optional | An internal description that you can attach to a payout. It can be useful for storing additional information about the customer in a structured format. You will be provided with these meta_data in your queries for payouts. Note: JSON structure is not enforced | Liu Tianwei
```

Comment:
Can we remove the `Note: JSON structure is not enforced` line? Might be confusing to some. If the developers want JSON they can put themselves, no issue

Reply:
@seowyanyi ok done, was thinking of addng it because in our meta_data tags in other APIs enforce the structure, you're right that it's better to not have it because they don't even need to know about such a thing since it's not enforced